### DISCUSSION TITLE
Handy bashrc aliasses to use with pipe-viewer.

### DISCUSSION THREAD

**Discussion by canisllup** [trizen/pipe-viewer#177] [Show and tell]
#Play the first result and then exit pipe-viewer (handy for stuff like checking movie trailer)
`p1='pipe-viewer --no-interactive --really-quiet --std-input='\''1'\'' -f'`

#Play continuous music (youtube radio)
`pr='pipe-viewer -a --no-interactive --quiet --std-input=1 --autoplay'`

**Comment by trizen** [comment]
Another variant of `pr` (without autoplay), would be:

```
pr2='pipe-viewer -n -A'
```

It plays all the results given a search keyword, including results from the next pages.
